stl/inc/header-units.json

```diff
@@ -112,7 +112,7 @@
         "typeinfo",
         "unordered_map",
         "unordered_set",
-        "use_ansi.h",

```

Comment:
This is pre-existing, but it might be better to skip these in the python code instead of commenting them out here, as comments are not valid in json (but are in json5, we could just say this is json5 and parsers need to support comments.

Reply:
> Fine for these early tests, but I really don't love the custom json comment handling.

Yeah, it's definitely a hack and I wish there were a built-in way to ignore comments.

> This is pre-existing, but it might be better to skip these in the python code instead of commenting them out here

The Python code could maintain a separate skip list. However, (with the exception of `version`, `yvals.h`, and `yvals_core.h`) the headers that are commented out in `header-units.json` need to be skipped by all users, not just the test harness, because those headers are variously incompatible with being treated as header units.

> as comments are not valid in json (but are in json5, we could just say this is json5 and parsers need to support comments.

By agreement with the compiler and build system teams, the format of `header-units.json` is the unofficial "JSON with comments" extension (I am not familiar with json5 but if it's valid in that format, that's good too - I assume that json5 is a superset so it will continue to remain valid).

Of course we could simply omit the ineligible headers, but then it would be difficult to see why they were missing (without a separate file explaining so), which is why I asked for comments to be allowed.